SRP/Example1.md

```diff
@@ -0,0 +1,22 @@
+### Usage
+
+---
+#### Where
+
+https://github.com/microsoft/vscode/blob/main/src/vs/editor/common/core/editOperation.ts
```

Comment:
➕ ➖
I have some doubts related to new Range inside the EditOperation. So basically the class not only edit but also create new instance of Range.